release/sbin/clear-all.sh

```diff
@@ -35,6 +35,26 @@ rm_dir() {
 if [[ ${OPENMLDB_MODE} == "standalone" ]]; then
   rm -rf standalone_db standalone_logs
 else
+  conf_file="conf/tablet.flags.template"
+  dirname=""
+  while IFS= read -r line
+  do
+    if echo "$line" | grep -q '^#'; then
+      continue
+    fi
+    if echo "$line" | grep -v "zk_root_path" | grep -q "root_path" ||
+        echo "$line" |  grep -q "openmldb_log_dir"; then
+      cur_dirname=$(echo "${line}" | awk -F '=' '{print $2}')
+      dirname="${dirname} ${cur_dirname}"
+    fi
+  done < "$conf_file"
+  for item in $dirname
```

Comment:
Don't do the dirty check. A common fix to that problem is double quoting the path, so `rm / abc` become `rm "/ abc"`. 
You can declare `dirname` as array, or manually concat with double quotes.